tests/test_bgp_neighbor_af.rb

```diff
@@ -407,7 +555,7 @@ def max_prefix(af, dbg)
     af.max_prefix_set(limit, threshold, warning)
     assert_equal(limit, af.max_prefix_limit,
                  "Test 4a. #{dbg} Failed to set limit to '#{limit}'")
-    assert_equal(threshold, af.max_prefix_threshold,
+    assert_equal((platform == :nexus ? threshold : 75), af.max_prefix_threshold,
```

Comment:
@glennmatthews It looks like threshold is not optional on XR (as it is on NX), 
or rather XR will nvgen the default threshold, whereas NX appears to have no (known) default threshold.

Setting the yaml default_value to 75 for XR should be fine, however the test should continue to work for either platform without modification.